src/peer.js

```diff
@@ -155,9 +155,6 @@ class Peer {
    * @returns {undefined}
    */
   close (callback) {
-    if (!this.conn || !this.stream) {
-      // no connection to close
-    }
```

Comment:
Why remove?

Reply:
Because it never invoked the callback, read the rest of the code.

This is still WIP, reviews are fine but DO NOT merge

Reply:
I take it back.  This code does nothing.